db/in_memory_stats_history.h

```diff
@@ -26,9 +38,17 @@ class InMemoryStatsHistoryIterator final : public StatsHistoryIterator {
   bool Valid() const override;
   Status status() const override;
 
+  // Move to the next stats snapshot currently available
+  // This function may invalidate the iterator
+  // REQUIRES: Valid()
   void Next() override;
+
+  // REQUIRES: Valid()
   uint64_t GetStatsTime() const override;
 
+  // This function is idempotent and will not be affected by the background
+  // garbage collection.
```

Comment:
I'm confused reading the comment. How would it be affected by the background GC?

Reply:
if it holds a pointer to the internal stored map? maybe just delete the part after "and"

Reply:
I feel that the description is unnecessary. Who will expect the map will can go away? How it is achieved perhaps should go to where the logic happened.